VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -39,8 +39,10 @@
             <LinearLayout
                 android:id="@+id/last_access_bar"
                 android:orientation="vertical"
-                style="@style/last_access_bar"
-                android:visibility="gone">
+                android:layout_width="match_parent"
+                android:layout_height="50dp"
+                android:background="@color/edx_brand_primary_x_light"
+                android:visibility="visible">
```

Comment:
Yes, this definitely needs to be `gone` by default.

![broke](https://cloud.githubusercontent.com/assets/1685613/12275726/ad18a7bc-b93f-11e5-8ab5-afdaa76d7b80.png)


If you'd like, you can also add `tools:visibility="visible"` so that you can still see it in the Android Studio preview.